### DISCUSSION TITLE
當代最保值物資—顯卡！

### DISCUSSION THREAD

**Discussion by ayakasanma** [1101-GS3073-C/Python-Community-GS3073-C#225] [General]
現代的顯卡簡直就是黃金，除了稀有以外甚至還會隨著時間漲價！在區塊鏈與AI大行其道的當下，顯卡價格到底有沒有可能回穩呢？讓我們繼續看下去～
這篇新聞是在講去年在美國被搶的顯卡竟流落到越南的黑市...比諜報片還離奇😂
https://www.4gamers.com.tw/news/detail/51587/stolen-evga-geforce-rtx-30-cards-reportedly-discovered-in-a-vietnamese-store

**Comment by ghost** [comment]
顯示卡真的是一張比一張還要貴XD真的很希望價格趕快回穩X
